src/admin-views/tribe-options-tickets.php

```diff
@@ -135,9 +135,15 @@
 		__( 'Events Tickets Plus', 'tribe-common' )
 	);
 
+	$plus_link_2 = sprintf(
+		'<a href="https://theeventscalendar.com/product/wordpress-event-tickets-plus/?utm_campaign=in-app&utm_medium=plugin-tickets&utm_source=post-editor" target="_blank">%s</a>',
+		__( 'Check it out!', 'tribe-common' )
+	);
```

Comment:
These links we already here, but please use the m.tri.be shortlinks
https://m.tri.be/new/

That way we can update there and not in the plugins. If there are related links this could be a good time to change them to m.tri.be